docs: improve bareos-webui documentation

bruno-at-bareos opened the pull request:
This PR aims to merge both chapter bareos-webui and bareos-webui-install(update).
With this move, end-user should find all webui related information in one place.

- merge BareosWebuiInstall.rst into BareosWebui.rst
- add instruction for AppArmor protected system
- Fix Bareos links redirection issues in WhatIsBareos.rst

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)


### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
- [x] Check backport line

joergsteffens reviewed (changes requested):
Looks good. I just found one typo. The other suggestions just remove information about outdated versions.
Maybe you can additonally remove section https://download.bareos.org/experimental/PR-1366/BareosMainReference/TasksAndConcepts/TransportEncryption.html#configuration-example-for-bareos-17-2 ?

And I'm unsure where the best place for this chapter will be. An alternative would be after the tuturial chapter.

joergsteffens commented:
The pr-tool complains:
```
 ✗  bareos-check-sources --since=47d9cdbd98cb28a0d6cabf2391b82884c6a35fd0 reported:
	Plugin 'trailing newlines' would modify 'docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst'
```
Also I think the commit messages should start with "docs:".

bruno-at-bareos commented on docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst:
```diff
@@ -4,6 +4,685 @@
+To make use of that feature in |webui|, a proper configured :config:option:`dir/profile/PluginOptionsACL` is required.
+
+
+Upgrade from 18.2.6 to 18.2.7
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Configuration changes
+^^^^^^^^^^^^^^^^^^^^^
+
+The configuration file :file:`configuration.ini` of the |webui| shipped with Bareos 18.2.7 introduced a new configuration
+parameter called ``filetree_refresh_timeout``. The default value is 120 seconds if not set explicitly.
+
+The |webui| triggers a Bvfs cache update automatically if required to be able to display the requested filetree.
+The configuration parameter has been introduced because in case of larger backup jobs with lots of files
+which are not already present in the Bvfs cache you could run into timeouts while trying to load the filetree
+in the restore module of the |webui|.
+
+If you have trouble with running into timeouts while loading the tree you can adjust the parameter ``filetree_refresh_timeout``
+to your needs. Keep in mind to set the timeout in your Apache or Nginx configuration accordingly to the setting
+in your :file:`configuration.ini`.
+
+In general we highly recommend updating the Bvfs cache frequently. Please see :ref:`section-updating-bvfs-cache-frequently` for further details on how to accomplish this.
+
```
Thanks I think so also.
@fbergkemper any through on that.

bruno-at-bareos commented:
I've fixed the new lines and also changed the PR subject to start with docs:
@joergsteffens shall I do a squash of commits before you go for green light?

joergsteffens commented:
Yes, please. What about my other two remarks from https://github.com/bareos/bareos/pull/1366#pullrequestreview-1287379777
Do you want to keep it the way it is?

bruno-at-bareos commented:
> Yes, please. What about my other two remarks from [#1366 (review)](https://github.com/bareos/bareos/pull/1366#pullrequestreview-1287379777) Do you want to keep it the way it is?

No, thanks for the reminders. At least I will remove the obsolete 17 part.

joergsteffens commented on docs/manuals/source/TasksAndConcepts/TransportEncryption.rst:
```diff
@@ -418,50 +418,6 @@ Please check the following configuration examples.  A complete table of the dire
-
-
-
 Configuration example for Bareos 18.2
```
```suggestion
Configuration example
```

joergsteffens replied:
And could you please also prepend the git commit message with "docs:" ? Or also squash them together.

bruno-at-bareos replied:
all done in last push.
